src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Cohort/DetailsViewModelMapperTests.cs

```diff
@@ -729,6 +729,21 @@ public async Task StatusIsMappedCorrectly_When_PendingApproval_From_TransferSend
             Assert.AreEqual("Pending - with funding employer", result.Status);
         }
 
+        [Test]
+        public async Task StatusIsMappedCorrectly_When_rejected_From_TransferSender()
+        {
+            var fixture = new DetailsViewModelMapperTestsFixture()
+                .CreateThisNumberOfApprenticeships(1)
+                .SetTransferSender()
+                .SetCohortWithParty(Party.TransferSender);
+
+            fixture.Cohort.IsApprovedByEmployer = fixture.Cohort.IsApprovedByProvider = true;
+            fixture.Cohort.TransferApprovalStatus = TransferApprovalStatus.Rejected;
```

Comment:
move this test setup into the test fixture

Reply:
Done